plugin/evm/block_builder_test.go

```diff
@@ -83,7 +83,7 @@ func TestBlockBuilderStopsTimer(t *testing.T) {
 	if builder.buildStatus != mayBuild {
 		t.Fatalf("expected build status to be %d but got %d", dontBuild, builder.buildStatus)
 	}
-	if !builder.isSC {
+	if !builder.isSE {
```

Comment:
should we update the test fatal message to be `isSE` as well?

Reply:
done